app/service/rest.py

```diff
@@ -644,11 +650,35 @@ def get_email_reply_to_address(service_id, reply_to_id):
     return jsonify(result.serialize()), 200
 
 
+@service_blueprint.route('/<uuid:service_id>/email-reply-to/verify', methods=['POST'])
+def verify_new_service_reply_to_email_address(service_id):
```

Comment:
Can we rename this to `verify_reply_to_email_address`?

Reply:
of course, good idea since we also use it to verify edited email addresses 💖